Functions/src/main/java/com/epam/engx/cleancode/functions/task4/Order.java

```diff
@@ -11,18 +11,21 @@
 
     public Double getPriceOfAvailableProducts() {
         double orderPrice = 0.0;
+        removeAvailableProduct();
```

Comment:
```suggestion
        removeNotAvailableProducts();
```
and there are `s` on the end needed